cmd/boulder-wfe2/main.go

```diff
@@ -45,6 +49,11 @@ type config struct {
 		SAService *cmd.GRPCClientConfig
 
 		Features map[string]bool
+
+		// CertificateChains maps AIA issuer URLs to certificate filenames.
+		// Certificates are read into the chain in the order they are defined in the
+		// slice of filenames.
+		CertificateChains map[string][]string
```

Comment:
nit: I kind of feel like this should be above the `Features` section, but for no real reason other than... it feels right?

Reply:
>  but for no real reason other than... it feels right?

Works for me :-)